providers/openlineage/docs/guides/user.rst

```diff
@@ -478,6 +478,51 @@ You can enable this automation by setting ``spark_inject_transport_info`` option
   AIRFLOW__OPENLINEAGE__SPARK_INJECT_TRANSPORT_INFO=true
 
 
+Passing parent information to Airflow DAG
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To enable full OpenLineage lineage tracking across dependent DAGs, you can pass parent and root job information
+through the DAG's ``dag_run.conf``. When a DAG run configuration includes an ``_openlineage`` section with valid metadata,
```

Comment:
Can we just make it `openlineage` without leading `_`?

Reply:
Do we ever would want user to fill this out themselves? If no, we can keep underscore. 

Would there be a mechanism in API or other external trigger mechanism to pass through this info as well?